samples/dymaptic.GeoBlazor.Core.Sample.TokenRefresh/dymaptic.GeoBlazor.Core.Sample.TokenRefresh.Client/Pages/Home.razor

```diff
@@ -0,0 +1,132 @@
+                                <a href="/map-auto" class="btn btn-primary btn-sm">View Auto Mode Demo</a>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-md-6">
+                        <div class="card border-success">
+                            <div class="card-body">
+                                <h6 class="card-title">💻 InteractiveWebAssembly Mode</h6>
+                                <p class="card-text small">Pure client-side execution. Best for rich interactivity and offline capability.</p>
+                                <a href="/map-webassembly" class="btn btn-success btn-sm">View WebAssembly Demo</a>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            
+            <div>
+                <p>
+                    Once your tokens are configured and working, navigate to the <strong>Map</strong> page 
+                    to see the fully authenticated GeoBlazor map in action! 
+                </p>
+                <ul>
+                    <li>Automatic token refresh keeps your map sessions active</li>
+                    <li>Secure server-side authentication protects your credentials</li>
+                    <li>Seamless integration with ArcGIS Portal or ArcGIS Online</li>
+                </ul>
+                <p>
+                    Ready to learn more? Check out the <a href="https://docs.geoblazor.com" target="_blank">GeoBlazor documentation</a> 
+                    and <a href="https://samples.geoblazor.com" target="_blank">samples</a> to unlock more possibilities!
+                </p>
+            </div>
+        </div>
+    </div>
+</div>
+
+@code {
+    private string _currentConfig = "";
+
+    protected override void OnInitialized()
+    {
+        // Show the template configuration as it appears in the sample appsettings.json
+        var configLines = new List<string>
+        {
+            "{",
+            "  \"Logging\": {",
+            "    \"LogLevel\": {",
+            "      \"Default\": \"Information\",",
+            "      \"Microsoft.AspNetCore\": \"Warning\"",
+            "    }",
+            "  },",
+            "  \"ArcGISAppId\": \"Your_ARCGIS_APP_ID\",",
+            "  \"ArcGISClientSecret\": \"Your_ARCGIS_CLIENT_SECRET\",",
+            "  \"ArcGISPortalUrl\": \"https://your-portal-url/portal/\",",
+            "  \"ArcGISAppTokenCacheFile\": \"arcgis_token_cache-file-name.json\",",
+            "  \"ArcGISTokenExpirationMinutes\": \"1440\",",
+            "  \"GeoBlazor\": {",
+            "    \"RegistrationKey\": \"YOUR_GEO_BLAZOR_REGISTRATION_KEY\"",
+            "  }",
+            "}"
+        };
+        _currentConfig = string.Join("\n", configLines);
```

Comment:
Hmm...I assumed this was actually reading the file, and would update when I changed the file. If we're going to hard code it, don't have it say "Current Server Configuration". Otherwise, have it read the file, or recreate it from `IConfiguration`.